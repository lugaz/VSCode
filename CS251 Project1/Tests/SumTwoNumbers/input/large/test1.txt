41959381062995603073597228868339081689152079675439741595883503296644712287130777378177538542145921809093577193053657401541583891398989992426722752518528354525919799950744781396452820636980901785809284395102184696469104184774342260042800123764073072604031640230568976169505191727926317782418713339466983792514584341006654642859682047884134033536043244474206819800513449531473996383984773043993234410269734422111830846359155102892734824643150890385077824919731856557717412024586551734869297251177483250506085607344457127122874639842573932467110749
759247748282379319559309000203351893308592399519477818843703567639173331235998092555970916709141984769752263416503109064651648649282424764767775049101460620858984264273584455340074455390736347072627178898320164878558214884845687284313146710799893582349242124133331330740861335167349077164046331773823472745269115707672999972721348751969096174325269713066361840989252691683992376013519288386145159389007105095750004818104157360554598066955671979385857346728371795541854386627349177185674002690393034177480890908972343186721636681057358634166772836829345613824897958393174157895718455354741087491399362479461893150331469505227377702492790695055091331590320307689245694534943723015980938412355543400216848868813972762747100072819777850741719749496552538802071460490101459111702736579303320589995248341938223987508253066510763299258757237402098327055385554691858104483297749282629381986716374280905876640806715861107272142917490959246206081317371083146706050267112937019431688663001749754941055260236
