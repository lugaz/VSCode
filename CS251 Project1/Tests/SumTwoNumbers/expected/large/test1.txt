759247748282379319559309000203351893308592399519477818843703567639173331235998092555970916709141984769752263416503109064651648649282424764767775049101460620858984264273584455340074455390736347072627178898320164878558214884845687284313146710799893582349242124133331330740861335167349077164046331773823472745269115707672999972721348751969096174325269713066361840989252691683992376013519288386145159389007105095750004818104157360554598066955671979385857388687752858537457460224578045524755691842472709617222486792475639831433923811834736811705314982751154707402091012050575699479609854344733514214151881007816419070131420250008774155313427675956877140874715409873942163639128497358240981212479307473289452900454203331723269578011505777059502168209892005785863975074442465766345596261351204724028784385182698194328053579960294773255141222175142320289795824426280216314144108437732274721541017431796261718631635592963829860329515545797940950614622260629956556352720281476558811537641592328873522370985
